server/api/files.go

```diff
@@ -123,37 +123,12 @@ func (a *API) handleServeFile(w http.ResponseWriter, r *http.Request) {
 	auditRec.AddMeta("teamID", board.TeamID)
 	auditRec.AddMeta("filename", filename)
 
-	fileInfo, err := a.app.GetFileInfo(filename)
+	fileInfo, fileReader, err := a.app.GetFile(board.TeamID, boardID, filename)
 	if err != nil && !model.IsErrNotFound(err) {
 		a.errorResponse(w, r, err)
 		return
 	}
 
-	if fileInfo != nil && fileInfo.Archived {
```

Comment:
Note to the reviewer- we no longer need this if block as this handled the file limits and we no longer have limits.